app/routes/git_tokens.py

```
@@ -4,340 +4,354 @@
-    description="Create a new git hosting service token configuration",
+	"/",
+	response_model=Dict[str, Any],
+	status_code=status.HTTP_201_CREATED,
+	summary="Add new git token",
+	description="Create a new git hosting service token configuration",
 )
 async def add_git_token(
-    request: Request,
-    payload: GitLabelCreate = Body(...),
-    current_user_id: str = Depends(get_current_user_id),
+		request: Request,
```

Pylint (3.3.7)

[warning] Unused argument 'request'

(W0613)